src/components/sidebar/nodeView/StopAreaView.tsx

```diff
@@ -330,6 +344,41 @@ class StopAreaView extends ViewFormBase<IStopAreaViewProps, IStopAreaViewState>
                             </div>
                         )}
                     </div>
+                    <div className={s.flexRow}>
```

Comment:
I think StopAreaView should be put into its own folder, then this whole table should be put into "stopAreaView/StopTable.tsx"

Our convention has been having each sidebar view in ints own folder, I think it has been a good decision